demosplan/DemosPlanCoreBundle/Logic/Procedure/ServiceStorage.php

```diff
@@ -1073,4 +1078,26 @@ private function handleTokenField($token): ?string
 
         return $token;
     }
+
+    private function validatePhaseIteration(array $procedure): array
```

Comment:
You could reduce the amount of returns. You could save the if conditions in variables. So you have two booleans -> one if condition, inside the if condition you set the error, and as default $mandatoryErrors is empty []. What do you think?

Reply:
I choose to use early return to be performant on this check, but you are right, this does'nt matter at this point.
Readability is more valuable here.
90368a8ef34c54b8bf90bcdb84a6e7fbed2b977d